workspace/__lib__/xod/bits/number-to-i32/patch.cpp

```diff
@@ -0,0 +1,17 @@
+
+struct State {
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    int32_t val = getValue<input_IN>(ctx);
+    uint8_t b3 = ((val >> 24) & 0xFF);
```

Comment:
I guess `& 0xFF` is not required here as the highest bits are always 0 after the shift.